README.md

```diff
@@ -8,22 +8,24 @@ This is intended to be the main driver for rendering [Scenic](https://github.com
 In your Nerves applications dependencies include the following line
 
     ...
-    {:scenic_driver_local, "~> 0.11"}
+    {:scenic_driver_local, github: "ScenicFramework/scenic_driver_local"}
     ...
 
+NOTE: For now you also have to use the scenic "input_and_drivers" branch
```

Comment:
This shouldn't be true any more. I merged all the support into v0.11

Reply:
Removed :+1: